demos/can-component/paginate.html

```diff
@@ -97,7 +97,7 @@
 	set.props.offsetLimit("offset", "limit")
 );
 
-superMap({
+baseMap({
```

Comment:
Switching to base-map because the can-fixture delay wasn’t perceivable; I think because the data was being fetched from local-storage before the fixture would resolve.